src/main/java/task/DeadlineTask.java

```diff
@@ -0,0 +1,32 @@
+package task;
+
+import exceptions.InvalidSyntaxException;
+import ui.Command;
+
+public class DeadlineTask extends Task {
+
+    // For serialization
+    private static final long serialVersionUID = (1 << 5) + 1;
```

Comment:
Remember to name your constants in all uppercase using underscore to separate words. Check for similar issue in other parts of the code.

Reply:
Thanks for pointing this out! Unfortunately, the field name is mandated by Java's `Serializable` interface which has been implemented by `Task` [(docs)](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/io/Serializable.html)